README.md

```diff
@@ -952,13 +952,43 @@ Presenter: _Daniel Miao_
 
 ## What's new in HealthKit
 
-https://developer.apple.com/wwdc20/`insert-session-number-here`
-
-Presenters: _Example Guy, Another Person_
-
-// TO-DO! You can contribute to this session, please see [CONTRIBUTING.md](CONTRIBUTING.md)
-
-
+https://developer.apple.com/wwdc20/10182/
+
+Presenters: _Netra Kenkarey_
+
+- **Symptoms**
+  - Developers can now track symptoms in HealthKit, read and write symptom samples
+  - HealthKit attempted to cover and track a wide range of symptoms (shortness of breath, sleep changes, appetite changes, fever, headache, fever etc)
+  - There are 13 symptom data types in HealthKit
+- **Electrocardiogram (ECG)**
+  - ECG samples will be available for reading in the latest version of iOS and watchOS 
+  - An ECG sample can be read as an [HKElectrocardiogram](https://developer.apple.com/documentation/healthkit/hkelectrocardiogram) (it represents a waveform as a series of voltage values)
+  - The [HKElectrocardiogram](https://developer.apple.com/documentation/healthkit/hkelectrocardiogram) sample is a collection of voltage measurements
+  - ECG sample has important properties that describe the measurements
+  - [classification (HKElectrocardiogram.Classification)](https://developer.apple.com/documentation/healthkit/hkelectrocardiogram/3551981-classification)
+    - Apple Watch will give the result of the recording in the form of a classification
+    - The classification is divided into 2 types
+      - Sinus Rhithm (heart is beating in a uniform pattern)
+      - Atrial fibrillation (form of irregular rhythm, user shopuld probably go see their doctor)
+    - If Apple Watch is unable to determine the ECG result, either due to a low or a high heart rate or due to any other reason, the result is considered inconclusive
+  - [symptomStatus (HKElectrocardiogram.SymptomsStatus)](https://developer.apple.com/documentation/healthkit/hkelectrocardiogram/3551984-symptomsstatus)
+    - It tells if the user associated a symptom with this ECG (e.g. heartburn, tightness in the chest)
+    - The symptom experienced can be recorded along with the ECG
+  - [averageHeartRage (HKQuantity?)](https://developer.apple.com/documentation/healthkit/hkelectrocardiogram/3551980-averageheartrate)
```

Comment:
Is it possible to format it like this? (not sure if it will display correctly)

```suggestion
  - [averageHeartRage (`HKQuantity?`)](https://developer.apple.com/documentation/healthkit/hkelectrocardiogram/3551980-averageheartrate)
```


If so, I'd recommend adding the backticks to all the Swift types above (e.g. `HKElectrocardiogram.Classification`, `HKElectrocardiogram.SymptomsStatus`, `HKQuantity?`, `Int`, etc) 🙏

Reply:
It works! :)